tests/dummy/public/test-forms/basic-fields/form.ttl

```diff
@@ -249,6 +249,17 @@ ext:conceptSchemeSelectorField a form:Field ;
 
 ext:mainFg form:hasField ext:conceptSchemeSelectorField .
 
+ext:conceptSchemeSelectorWithOptionPredicatesField a form:Field ;
+    sh:name "Select with form:options as separate predicates" ;
+    sh:order 270 ;
+    sh:path ext:conceptSchemeSelectorValue ;
+    ext:conceptScheme "http://example-concept-schemes/concept-schemes/foo-bar-baz" ;
```

Comment:
Thinking to make it possible to just pass it as prefix:predicate instead of the string. This is done like this atm as the json options handled it as a string